pyfat/get_channels.py

```diff
@@ -0,0 +1,154 @@
+import pandas as pd
+
```

Comment:
Since I dont know how your data is formatted, I am going to assume this is all good!